### DISCUSSION TITLE
NixOS module

### DISCUSSION THREAD

**Discussion by CaptainJawZ** [IgnisDa/ryot#800] [General]
Hello! I've been working on the packaging of ryot for nix, and with the resent addition of 5.0 and nixos 24.05 around the corner, figured out it's a good time to put some energy into it, I was wondering if I can ask you some questions that come to mind during the process through here?

Right now my only question is if PostgreSQL is the only supported database, i saw that mysql support got dropped, right?

**Comment by IgnisDa** [comment]
Yes, we used to support MySQL and sqlite but it was really difficult to maintain all 3, so i dropped them. Now only postgres is supported.

If you decide to add support, I'd appreciate if you make a PR to the deployment docs as well.

**Reply by CaptainJawZ** [reply]
I'll think about it, I'm not the best at MySQL but I could give a crack with sqlite, no promises lol.

for now I'll focus on getting this nicely well integrated with nixos!

**Reply by IgnisDa** [reply]
I think I might have miscommunicated. Ryot ONLY supports postgres and no other database backend.
